assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -35,17 +35,3 @@
   }
 }
 
-.component-stripe-pop-up-button--disable {
```

Comment:
FYI, you can use `.component-stripe-pop-up-button:disabled` to get the same result